frame/staking/src/lib.rs

```diff
@@ -686,6 +686,8 @@ impl Default for Forcing {
 enum Releases {
 	V1_0_0Ancient,
 	V2_0_0,
+	V3_0_0,
+	V4_0_0,
 }
 
 impl Default for Releases {
```

Comment:
I don't know what to do with this default releases.
If the storage is empty then it means either that the storage version is before the first runtime_upgrade that introduced this storage, or that this module has been introduced in a running chain (without building its genesis config).